pygmt/figure.py

```diff
@@ -228,8 +228,8 @@ def psconvert(self, **kwargs):
             towards black (100%) [no fading, 0]. Append **+g**\ *paint* to
             paint the BoundingBox behind the illustration and append **+p**\
             [*pen*] to draw the BoundingBox outline (append a pen or accept
-            the default pen of 0.25p,black). **Note**: If both **+g** and
-            **+f** are used then we use paint as the fade color instead of
+            the default pen of ``"0.25p, black, solid"``). **Note**: If both **+g** and
```

Comment:
Whitespaces are not allowed here.
```suggestion
            the default pen of ``"0.25p,black,solid"``). **Note**: If both **+g** and
```